### DISCUSSION TITLE
Enable replies on questions in Q&A format

### DISCUSSION THREAD

**Discussion by john-hen** [community/community#9600] [Discussions]
In discussions of the Q&A format, we can "write a reply" underneath any of the answers, but we cannot do the same for the question itself. However, some questions need to be clarified by their author to make them answerable. What this leads to, is people "suggesting answers" that are actually just requests for clarification, and not actual attempts to answer the question. Or people may just ignore the question because it cannot be answered as written. Both scenarios lessen the value of the Q&A format.

I would suggest to enable the reply-like functionality for the question post as well. I say "reply-like" because that terminology might be confusing in that context. (Or even generally: reply and answer often mean the same thing. I'd call this a comment, but that term is already used for posts in the open-ended discussion format.)

**Comment by DanielPower** [comment]
I'm trying to propose GitHub discussions at my company, and I'm having the same problem. Often when a question is asked, it cannot be immediately answered without a little bit of back and forth. 

The current UX doesn't allow that. As somebody reading a question post, your options are to either provide an answer, or ignore it. Or to "provide an answer" which is not actually an answer. Then start a discussion in that thread. If an answer comes up from that thread, copy/paste it as another answer?

It's confusing at best.

PS: This post is not an answer. But it's an answer.
